app/models/katello/glue/pulp/repo.rb

```diff
@@ -744,6 +744,13 @@ def ostree?
         self.content_type == Repository::OSTREE_TYPE
       end
 
+      def capsule_download_policy
```

Comment:
ohh, so we could just do:

```
self.download_policy if self.respond_to?(:download_policy)
```

As you can tell, the check on it being Katello::Repository object keeps hanging me up :)